library/src/wu/seal/jsontokotlin/JsonToKotlinBuilder.kt

```diff
@@ -0,0 +1,209 @@
+package wu.seal.jsontokotlin
+
+import extensions.chen.biao.KeepAnnotationSupport
+import extensions.jose.han.ParcelableAnnotationSupport
+import extensions.ted.zeng.PropertyAnnotationLineSupport
+import extensions.wu.seal.*
+import extensions.xu.rui.PrimitiveTypeNonNullableSupport
+import wu.seal.jsontokotlin.test.TestConfig
+
+
+/**
+ * To get Kotlin class code from JSON
+ */
+class JsonToKotlinBuilder() {
+
+    init {
+        TestConfig.apply {
+
+            isTestModel = true
+            isCommentOff = false
+            isOrderByAlphabetical = true
+            isPropertiesVar = false
+            targetJsonConvertLib = TargetJsonConverter.NoneWithCamelCase
+            propertyTypeStrategy = PropertyTypeStrategy.AutoDeterMineNullableOrNot
+            defaultValueStrategy = DefaultValueStrategy.None
+            isNestedClassModel = true
+            customPropertyAnnotationFormatString = "@Optional\n@SerialName(\"%s\")"
+            customAnnotaionImportClassString = "import kotlinx.serialization.SerialName\n" +
+                    "import kotlinx.serialization.Serializable" + "\n" + "import kotlinx.serialization.Optional"
+            customClassAnnotationFormatString = "@Serializable"
+            enableMinimalAnnotation = false
+            indent = 4
+            parenClassTemplate = ""
+            isKeywordPropertyValid = true
+            extensionsConfig = ""
+        }
+    }
+
+    /**
+     * To set property type to `var`, pass true.
+     * Default : false
+     */
+    fun setPropertiesVar(isVar: Boolean): JsonToKotlinBuilder {
+        TestConfig.isPropertiesVar = isVar
+        return this
+    }
+
+    /**
+     * To set if the properties can be null or not
+     * Default: PropertyTypeStrategy.NotNullable
+     */
+    fun setPropertyTypeStrategy(strategy: PropertyTypeStrategy): JsonToKotlinBuilder {
+        TestConfig.propertyTypeStrategy = strategy
+        return this
+    }
+
+    /**
+     * To set default value.
+     * Default : DefaultValueStrategy.AvoidNull
+     */
+    fun setDefaultValueStrategy(strategy: DefaultValueStrategy): JsonToKotlinBuilder {
+        TestConfig.defaultValueStrategy = strategy
+        return this
+    }
+
+    /**
+     * To set JSON decoding/encoding library
+     * Default: TargetJsonConverter.Gson
```

Comment:
I think the default value should be `TargetJsonConverter.None`, Keep the same with plugin

Reply:
It's `None'. Need to change just the comment. Will do it.